README.markdown

```diff
@@ -423,18 +424,18 @@ It is also extremely appreciated to use comments to separate the [Class Attribut
 
 **Preferred:**
 ```swift
-class MyViewController                      : UIViewController {
+class MyViewController                           : UIViewController {
 
     // MARK: - Outlets
 
-    @IBOutlet var lettersGameButton         : UIButton?
-    @IBOutlet var headBodyLegsGameButton    : UIButton?
-    @IBOutlet var colorMatcherGameButton    : UIButton?
+    @IBOutlet weak var lettersGameButton         : UIButton?
+    @IBOutlet weak var headBodyLegsGameButton    : UIButton?
+    @IBOutlet weak var colorMatcherGameButton    : UIButton?
 
```

Comment:
We do.

"Swift properties are strong by default. Use the weak keyword to indicate that a property has a weak reference to the object stored as its value. This keyword can be used only for properties that are optional class types. For more information, see Attributes". --https://developer.apple.com/library/ios/documentation/Swift/Conceptual/BuildingCocoaApps/WritingSwiftClassesWithObjective-CBehavior.html